rover-server/README.md

```diff
@@ -0,0 +1,10 @@
+<!--
+Copyright 2025 Deutsche Telekom IT GmbH
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# (legacy) Rover-Server 
```

Comment:
Updateing the readme would be nice. no need for full-fledged diagrams as the others. but perhaps to cover
* why rover server is needed
* why the inputs from example don't align 1:1 with `rover-domain`